lein-jank/test-project/project.clj

```diff
@@ -0,0 +1,7 @@
+(defproject test-project "0.1.0-SNAPSHOT"
+  :description "FIXME: write description"
+  :url "http://example.com/FIXME"
+  :license {:name "EPL-2.0 OR GPL-2.0-or-later WITH Classpath-exception-2.0"
+            :url "https://www.eclipse.org/legal/epl-2.0/"}
```

Comment:
Let's remove these lines, rather than updating the license to MPL (what jank uses).